06-building-a-program-start-to-finish/writing-tests/output/inverted-index-test.js

```diff
@@ -0,0 +1,146 @@
+/* eslint-disable */
+const books = require('../books.json');
+const adventureBooks = require('../adventure-books.json');
+const emptyArray = require('../empty.json');
+const invalidContent = require('../invalid-content.json');
+const invalidFile = require('../invalid-file.json');
+const invalidKey = require('../invalid-key.json');
+/* eslint-enable */
+
+const index = new InvertedIndex();
+
+describe('Test for valid book data', () => {
+    it('Should have keys named \'title\' and \'text\' with string for values', () => {
```

Comment:
I'll stop the review of this file right here because it's obvious you're not using the style guides at all. Kindly fix and raise another PR.